frontend/dockerfile/instructions/commands.go

```diff
@@ -367,6 +380,15 @@ func (c *ArgCommand) Expand(expander SingleWordExpander) error {
 	return nil
 }
 
+func (c *ArgCommand) BuildKeyValuePairOptional(values map[string]string) KeyValuePairOptional {
```

Comment:
I think this makes better sense as a helper method in dockerfile2llb package. `setKVValue(KeyValuePairOptional, map[string]string)`.